.github/actions/prepare-ios-simulator/action.yml

```diff
@@ -1,28 +0,0 @@
-name: 'Prepare iOS Simulator'
-inputs:
-  device:
-    description: 'The device name'
-    required: true
-  ios:
-    description: 'The iOS version'
-    required: true
-  xcode:
-    description: 'The Xcode version. The ios simulators supported for each xcode version: https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#installed-simulators'
-    required: true
-runs:
-  using: "composite"
-  steps:
-    - run: |
-        xcode_version=${{ inputs.xcode }}
-        ios_version=${{ inputs.ios }}
-        ios_version_dash=${ios_version//./-} # ex: 12.4 -> 12-4
-
-        sudo mkdir -p /Library/Developer/CoreSimulator/Profiles/Runtimes
-
-        sudo ln -s /Applications/Xcode_$xcode_version.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime /Library/Developer/CoreSimulator/Profiles/Runtimes/iOS\ $ios_version.simruntime
-
-        xcrun simctl list runtimes
-        xcrun simctl create custom-test-device "${{ inputs.device }}" "com.apple.CoreSimulator.SimRuntime.iOS-$ios_version_dash"
-        xcrun simctl list devices $ios_version
-      shell: bash
-  
```

Comment:
Why we don't need this anymore?

Reply:
It was a legacy to save time on CI to configure the simulator. Right now everything works out of the box on Github Actions.